bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-dashboard",
-  "version": "0.0.1",
+  "version": "1.2.1",
```

Comment:
Why the `0.0.1` → `1.2.1` leap?

Reply:
Yeah, this is very question actually!
Version number is managed by tags only in this repo and current version is **1.2.0** (reference is available here https://github.com/jobready/neptune/blob/develop/Bowerfile#L11) but I was surprised that version wasn't managed in original `bower.js` or `package.json` and I decided to actualize versions in that files and repo tags. If we don't need it, I'm happy to rollback these changes 👍